generator/src/main/java/com/scottlogic/deg/generator/outputs/datasetwriters/JsonFormatter.java

```diff
@@ -0,0 +1,28 @@
+package com.scottlogic.deg.generator.outputs.datasetwriters;
+
+import com.scottlogic.deg.generator.outputs.GeneratedObject;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+
+public class JsonFormatter implements RowOutputFormatter<String> {
```

Comment:
I don't think we should be writing our own JSON serialisation - there's a lot of complexity in doing that in a spec-compliant and performant fashion.